applications/crossbar/src/modules/cb_system_configs.erl

```diff
@@ -59,9 +61,17 @@ init() ->
 -spec authorize(cb_context:context()) -> boolean().
 -spec authorize(cb_context:context(), path_token()) -> boolean().
 -spec authorize(cb_context:context(), path_token(), path_token()) -> boolean().
-authorize(Context) -> cb_context:is_superduper_admin(Context).
-authorize(Context, _Id) -> cb_context:is_superduper_admin(Context).
-authorize(Context, _Id, _Node) -> cb_context:is_superduper_admin(Context).
+authorize(Context) ->
+    cb_context:is_superduper_admin(Context)
+        andalso
+        is_system_request(Context).
+authorize(Context, _Id) -> authorize(Context).
+authorize(Context, _Id, _Node) -> authorize(Context).
+
+-spec is_system_request(cb_context:context()) -> boolean().
+is_system_request(Context) ->
+    [{Mod, _}|_] = cb_context:req_nouns(Context),
```

Comment:
@lazedo, why?
Nouns are of type
-type cb_mod_with_tokens() :: {ne_binary(), path_tokens()}.
-type cb_mods_with_tokens() :: [cb_mod_with_tokens()].